modules/n1ql/pages/n1ql-language-reference/dropsequence.adoc

```diff
@@ -0,0 +1,122 @@
+= DROP SEQUENCE
+:page-topic-type: reference
+:page-status: Couchbase Server 7.6
+:imagesdir: ../../assets/images
+:description: The DROP SEQUENCE statement enables you to drop a sequence in a given keyspace.
+
+// Cross-references
+:authorization-overview: xref:learn:security/authorization-overview.adoc
+:query-context: xref:n1ql:n1ql-intro/queriesandresults.adoc#query-context
+:identifiers: xref:n1ql-language-reference/identifiers.adoc
+:logical-hierarchy: xref:n1ql-intro/sysinfo.adoc#logical-hierarchy
+
+// TEMP
+include::partial$n1ql-language-reference/horizontal-style.adoc[]
+
+[abstract]
+{description}
+
+== Purpose
+
+include::./sequenceops.adoc[tags=overview]
+
+== Prerequisites
+
+.RBAC Privileges
+To execute the DROP SEQUENCE statement, you must have the _Query Manage Sequences_ privilege granted on the scope.
+For more details about user roles, see {authorization-overview}[Authorization].
+
+== Syntax
+
+[source,ebnf]
+----
+include::partial$grammar/ddl.ebnf[tag=drop-sequence]
+----
+
+image::n1ql-language-reference/drop-sequence.png["Syntax diagram: refer to source code listing", align=left]
+
+// TODO: Automatic links in EBNF.
+
+[horizontal]
+sequence:: (Required) A name that identifies the sequence within a namespace, bucket, and scope.
+See <<sequence>> below.
+
+[[sequence]]
+=== Sequence Name
+
+[source,ebnf]
+----
+include::partial$grammar/ddl.ebnf[tag=sequence]
+----
+
+image::n1ql-language-reference/sequence.png["Syntax diagram: refer to source code listing", align=left]
+
+The sequence name specifies the name of the sequence to drop.
+
+Each sequence is associated with a given namespace, bucket, and scope.
+You must specify the namespace, bucket, and scope to refer to the sequence correctly.
```

Comment:
This may be done via the query_context - i.e. doesn't have to be explicitly stated in every statement.  Namespace "default" is implied when not specified (same as for keyspace references etc.).